themes/sutro/template.html

```diff
@@ -0,0 +1,692 @@
+<!-- Sutro -->
+<style>
+  media-controller {
+    --base: 16px;
+    min-width: 300px;
+
+    font-size: calc(0.75 * var(--base));
+    font-family: Roboto, Arial, sans-serif;
+    --media-font-family: Roboto, helvetica neue, segoe ui, arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+
+    --media-primary-color: #fff;
+    --media-secondary-color: transparent;
+    --media-menu-background: rgba(28, 28, 28, 0.9);
+    --media-control-hover-background: var(--media-secondary-color);
+
+    --media-range-track-height: calc(0.125 * var(--base));
+    --media-range-thumb-height: var(--base);
+    --media-range-thumb-width: var(--base);
+    --media-range-thumb-border-radius: var(--base);
+
+    --media-preview-thumbnail-border: calc(0.125 * var(--base)) solid #fff;
+    --media-preview-thumbnail-border-radius: calc(0.125 * var(--base));
+    --media-preview-time-margin: calc(0.25 * var(--base)) 0 0;
+  }
+
+  media-controller[breakpointmd] {
+    --base: 20px;
+  }
+
+  /* The biggest size controller is tied to going fullscreen
+      instead of a player width */
+  media-controller[mediaisfullscreen] {
+    --base: 28px;
+  }
+
+  .media-button {
+    position: relative;
+    display: inline-block;
+    width: calc(2 * var(--base));
```

Comment:
I wouldn't restrict the button width I think, that's the one that is variable. better resize on the height.

also there is a CSS var `--media-control-height` that might be handy for this.

<img width="368" alt="SCR-20240801-kumt" src="https://github.com/user-attachments/assets/19b14214-ecd8-4c55-bcbe-61fa7cba1915">